web/src/components/speaker/SpeakerCard.vue

```diff
@@ -35,12 +35,15 @@
               >
                 {{ $t('findSpeaker.pronouns', [pronouns]) }}
               </div>
-              <p class="pa-0 ma-0">{{ speaker.get('speaker_bio') }}</p>
+              <p class="pa-0 ma-0">
```

Comment:
Do you have prettier or some other formatter moving things here? (Should we all use prettier? I haven't used it much and have mostly relied on eslint)

Reply:
I don't think I have prettier, I have been running `yarn lint --fix` which uses the project configuration and not the editor one (I disabled auto-formatting on my editor) 🤔